eth-contracts/contracts/service/ServiceProviderFactory.sol

```diff
@@ -22,14 +22,17 @@ contract ServiceProviderFactory is RegistryContract {
     }
 
     mapping(bytes32 => ServiceInstanceStakeRequirements) serviceTypeStakeRequirements;
-    // END Temporary data structures
 
     // Maps directly staked amount by SP, not including delegators
     mapping(address => uint) spDeployerStake;
 
     // % Cut of delegator tokens assigned to sp deployer
     mapping(address => uint) spDeployerCut;
 
+    // Bool indicating whether this SP has met the min/max stake requirements
+    mapping(address => bool) spValidBounds;
```

Comment:
should we consider combining bounds, cut and stake into a single struct...?

Reply:
again - this can be broken up into a separate PR, and doesn't have to be handled here